src/api/app/views/webui2/webui/project/side_links/_requests.html.haml

```diff
@@ -9,4 +9,8 @@
     end
   %i.fa.fa-info-circle.text-info
   = link_to(path) do
-    = pluralize(requests.size, 'open request')
+    = pluralize(incoming_requests.size, 'Incoming request' )
+  %br
```

Comment:
Instead of using a `<br>`, these links should be in list items (`<li>`), so this partial should be:

```
%li
  link for incoming requests
%li
  link for outgoing requests
```